scripts/find_cord19_length_outliers.py

```diff
@@ -0,0 +1,68 @@
+from pyserini.collection import pycollection
+import pandas as pd
+
+
+
+if __name__ == '__main__':
+    titles_withText = []
+    abstracts_withText = []
```

Comment:
`_with_text` to be consistently pythonic?

Reply:
Better yet, why not a dictionary?

Reply:
I updated with a dictionary